doc/hardware/porting/board_porting.rst

```diff
@@ -912,9 +912,69 @@ There are some extra things you'll need to do:
 #. Prepare a pull request adding your board which follows the
    :ref:`contribute_guidelines`.
 
+.. _extend-board:
+
 Board extensions
 ****************
 
+The board hardware model in Zephyr allows you to extend an existing board with
+new board variants. Such board extensions can be done in your custom repository
+and thus outside of the Zephyr repository.
+Zephyr's board hardware model allows you to extend existing boards with new
+variants. Such board extensions can be done in your custom repository and thus
+outside of the Zephyr repository.
```

Comment:
repeated. I tend to like second paragraph's formulation better, fwiw
also would recommend skipping a line so that "Extending an existing board with [...]" is another paragraph

Reply:
of course you like the second paragraph best, cause you wrote it yourself :wink: 
I happened to just apply your suggestion in this comment https://github.com/zephyrproject-rtos/zephyr/pull/73409#discussion_r1636802273 using the `commit suggestion` feature of github.

Guess I should have noticed that you were actually not removing the old lines in that suggestio :laughing: 

Repeated paragraph removed.